### DISCUSSION TITLE
source compiles, programs, then teensy bootloops

### DISCUSSION THREAD

**Discussion by AtomicRPM** [frntc/SIDKick#6] [General]
I downloaded and compiled you source and all it does is boot loop on the teensy.  Any suggestions?

**Comment by frntc** [comment]
Hm, no idea. I used the pretty old Arduino-1.8.13, maybe that makes a difference?

**Comment by AtomicRPM** [comment]
Thanks.  I completely overlooked that in the readme.   I ended up installing everything on an old laptop.   I tried on my on linux desktop and it appears the two versions of the IDE don't play nice when installed together on the same machine.  1.8.12 still used the latest libraries.  This issue resolved!  Thanks, again!
